app/javascript/components/layout/CustomTable.js

```diff
@@ -203,7 +203,14 @@ class CustomTable extends React.Component {
           </div>
         )}
         <div className={this.props.tableCustomClass ? `table-responsive ${this.props.tableCustomClass}` : 'table-responsive'}>
-          <Table striped bordered hover size="sm" id={`${this.props.dataType}-table`} className="opaque-table">
+          <Table
+            striped
+            bordered
+            hover
+            size="sm"
+            id={`${this.props.dataType}-table`}
+            className="opaque-table"
+            title={this.props.title ? this.props.title : null}>
```

Comment:
This `title` allows screen readers to read "Entering Close Contact Table" instead of reading "Entering Table" (as is had been doing)